app/globals.css

```
@@ -135,8 +135,11 @@
     @apply border-border outline-ring/50;
   }
 
+  html,
   body {
     @apply bg-background text-foreground;
+    overflow: hidden;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)